homeassistant/components/yolink/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "yolink",
+  "name": "YoLink",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/yolink",
+  "requirements": ["yolink-api==0.0.3"],
```

Comment:
Please publish the library on github.com besides on PyPI. I don't see a link to a github repo on the PyPI page:
https://pypi.org/project/yolink-api/

Reply:
Thanks! Please also publish github releases that correspond to the releases on PyPI.

I recommend adding a github action to automate this.

Example:
https://github.com/home-assistant-libs/aiowebostv/blob/67fd704abcf66afe165c3e244304ec7d421305ca/.github/workflows/publish-to-pypi.yml#L1-L27

Reply:
Thanks! i created workflow, but pypi repo has yolink-0.0.4 alreay, so i release it by myself.

Reply:
Great!